bph_pal/bph_pal/__main__.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2019 Kevin Weiss, for HAW Hamburg  <kevin.weiss@haw-hamburg.de>
+#
+# This file is subject to the terms and conditions of the MIT License. See the
+# file LICENSE in the top level directory for more details.
+# SPDX-License-Identifier:    MIT
+"""packet init for BPH PAL
+This exposes a main function to run the packet from CLI
+"""
+import argparse
+import logging
+
+
+try:
+    from .bph_shell import BphShell as BPH
```

Comment:
this whole try except looks ugly, I bet there is a nice way (I look into it)

Reply:
I agree, I would love a 
`from .bph_shell or bph_pal.bph_shell or bph_shell import BphShell as BPH` but I couldn't find anything that does that yet.

Reply:
look at https://github.com/riot-appstore/PHiLIP/blob/master/IF/philip_pal/philip_pal/__main__.py ??